kubernetes/config/incluster_config.c

```diff
@@ -65,13 +65,24 @@ static int setBasePathInCluster(char **pBasePath)
     }
 
     int basePathSize = strlen(SERVICE_HTTPS_PREFIX) + strlen(service_host_env) + strlen(service_port_env) + 2 /* 1 for ':', 1 for '\0' */ ;
+    bool isIPv6 = false;
+    if (strchr(service_host_env, ':') == NULL)
```

Comment:
I think this is the case for IPv4 (cannot find '`:`' in `service_host_env` )

Reply:
Right, good catch. thanks!